test/stdlib/Result.swift

```diff
@@ -0,0 +1,186 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+import Swift
+
+let ResultTests = TestSuite("Result")
+
+fileprivate enum Err: Error, Equatable {
+    case err
+    case derr
+}
+
+fileprivate let string = "string"
+
+ResultTests.test("Construction") {
+    let result1: Result<String, String> = .success(string)
```

Comment:
Nit: Can you re-indent this file at 2 spaces?

Reply:
Done.